Metro-Records/la-metro-councilmatic
# Add 2024 event calendar

## Overview

See title.

Connects #965 

### Demo

<img width="1512" alt="Screenshot 2023-05-16 at 1 35 29 PM" src="https://github.com/Metro-Records/la-metro-councilmatic/assets/12176173/c0d894df-b969-4a46-a7e1-464fa96182f0">

<img width="1512" alt="Screenshot 2023-05-16 at 1 35 12 PM" src="https://github.com/Metro-Records/la-metro-councilmatic/assets/12176173/3a7a30dc-9017-4ce3-8c58-f3b21defba60">

## Testing Instructions

 * View the homepage and confirm the event calendar links look good and resolve correctly.
 * Ditto for the event listing page.
